tests/inverted-index-testSpec.js

```diff
@@ -191,28 +223,28 @@ describe('Inverted Index Tests', () => {
     for (const individualQuery of allQuery) {
       allUniqueQuery.push(individualQuery);
     }
-    it(`Should split on whitespaces[single or multiple] and tabs, and filter out symbols 
-    from text whenever sanitize data is called in a method`, () => {
-      expect(allUniqueQuery).toEqual(['first', 'string', 'around', 'world', 'remincense']);
+    it('split on whitespaces and tabs, and filter out symbols from text', () => {
+      expect(allUniqueQuery)
+      .toEqual(['first', 'string', 'around', 'world', 'remincense']);
     });
   });
 
   describe('Other testcases [preventing code breaking]', () => {
-    const newInvertedIndex = new newIndex();
-    const allfiles = ['book-one.json', 'invalid-file.json', 'empty-book.json', 'book-three.json'];
+    const newInvertedIndex = new newIndex(),
```

Comment:
It is testing to ensure that if an invalid file is uploaded alongside a valid one one, the app does not crash because of the invalid file. In the allfiles variable, there are two valid files that is being passed into the function along with two invalid files.